kDrive/UI/Controller/Menu/SecurityTableViewController.swift

```diff
@@ -74,7 +74,25 @@ class SecurityTableViewController: BaseGroupedTableViewController {
             cell.titleLabel.text = KDriveResourcesStrings.Localizable.fileProviderExtensionTitle
             cell.detailsLabel.text = KDriveResourcesStrings.Localizable.fileProviderExtensionDescription
             cell.switchHandler = { sender in
-                UserDefaults.shared.isFileProviderExtensionEnabled = sender.isOn
+                let toggleStatus = sender.isOn
+                cell.valueSwitch.isEnabled = false
```

Comment:
Not in love with doing this way, but it does the job. On re-enable it is necessary given I do a some network calls (if many accounts)

Reply:
Do we really need to do a network call ? 
If we are in this settings, Accounts / Drives should be up to date

We could simply take the drives we have for each user and add the domains.

(We can keep the Task because this process is still async but not network dependant)

Reply:
replied to this